go/billing/templatetags/billing_tags.py

```diff
@@ -3,32 +3,42 @@
 from django import template
 from django.utils.translation import ungettext
 
+from go.base.utils import format_currency
 from go.billing import settings
 
 register = template.Library()
 
 
+@register.filter
+def format_cents(v):
+    """returns a formatted string representation of a number in dollars from a
+    decimal cents value (cents are the billing system's internal representation
+    for monetary amounts).
+    """
+    v = v / Decimal('100.0')
```

Comment:
Hmm, it looks like the `LineItem` model's `credits` field is an integer field, weird :/ will have to fix that before this can land.